canonical/ubuntu.com
# [WD-7442] update chart and table of Ubuntu releases on /about page

## Done

- Actualized data shown in the chart and in the table of "Ubuntu Releases" section of /about page
- Removed another version of releases chart/table and show only one version of it on both /about and /about/release-cycle pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Navigate to /about and check the releases chart on big screens and the table on mobile screens.
   - It should not show releases that are not supported anymore
- Navigate to /about/release-cycle and check that it shows the same table/chart as in /about page

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-7442

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
